projects/1.21/assets/psi/psi/lang/zh_cn.json

```
@@ -0,0 +1,1123 @@
+  "psi.spellerror.armor": "错误：选择符只可在盔甲内施展。",
+  "psi.spellerror.no_message": "错误：CAD尚未获取到广播信息。",
+  "psi.spellerror.comparator": "错误：比较器只可从4个正方向上测量信号。",
+  "psi.spellerror.cad_casting_only": "错误：此启动式必须由CAD施展。",
+  "psi.spellerror.nan": "错误：%s参数不是数值。",
+  "command.psi.learn.usage": "/psi-learn <组别> [玩家]",
+  "command.psi.learn.success": "%s成功习得了%s",
+  "command.psi.learn.success.all": "%s成功习得了Psi的全部知识",
+  "command.psi.learn.should_not": "%s已经知晓了%s",
+  "command.psi.learn.players": "%s不是玩家，只有玩家能学习Psi知识",
+  "command.psi.learn.console": "终端无法学习Psi知识，请指定一名玩家！/psi-learn <组别> [玩家]",
+  "command.psi.learn.not_a_group": "“%s”不是有效的Psi功能块组",
+  "command.psi.learn.unknown": "这名玩家没有Psi数据。请向模组作者反馈。",
+  "command.psi.unlearn.usage": "/psi-unlearn <组别> [玩家]",
+  "command.psi.unlearn.success": "%s成功遗忘了%s",
+  "command.psi.unlearn.success.all": "%s成功遗忘了Psi的所有知识",
+  "command.psi.unlearn.should_not": "%s尚未知晓%s",
+  "command.psi.unlearn.players": "%s不是玩家，只有玩家能学习Psi知识",
+  "command.psi.unlearn.console": "终端无法学习Psi知识，请指定一名玩家！/psi-unlearn <组别> [玩家]",
+  "command.psi.unlearn.not_a_group": "“%s”不是有效的Psi功能块组",
+  "command.psi.unlearn.unknown": "这名玩家没有Psi数据。请向模组作者反馈。",
+  "psi.tutorial1": "教程1",
+  "psi.tutorial2": "教程2",
+  "psi.tutorial3": "教程3",
+  "psi.tutorial4": "教程4",
+  "psi.numbers_intro": "数值101",
+  "psi.vectors_intro": "矢量101",
+  "psi.entities_intro": "实体101",
+  "psi.projectiles": "变种施术",
+  "psi.block_works": "方块工程",
+  "psi.infusion": "Psi金属灌注",
+  "psi.movement": "高级移动",
+  "psi.loopcasting": "循环施术",
+  "psi.block_movement": "方块移动",
+  "psi.elemental_arts": "元素技艺",
+  "psi.greater_infusion": "高级灌注",
+  "psi.tool_casting": "工具施术",
+  "psi.positive_effects": "正面效果",
+  "psi.negative_effects": "负面效果",
+  "psi.exosuit_casting": "装甲施术",
+  "psi.trignometry": "三角学",
+  "psi.smeltery": "烧炼",
+  "psi.flow_control": "流程控制",
+  "psi.block_conjuration": "方块构筑",
+  "psi.eidos_ceversal": "情报体操纵",
+  "psi.detection_dynamics": "探测与焦点",
+  "psi.memory_management": "内存管理",
+  "psi.secondary_operators": "次级运算符",
+  "psi.misc_tricks": "杂项启动式",
+  "psi.list_operations": "列表操作",
+  "psi.fake_level_psidust": "Psi粉末灌注",
+  "jei.psi.category.trick": "法术灌注",
+  "jei.psi.spell_copy": "复制术式",
+  "psi.subtitle.book": "Alexa：播放音乐",
+  "_comment": "Book stuff",
+  "advancement.psi:iron_cad_pickup": "棱角分明",
+  "advancement.psi:iron_cad_pickup.desc": "合成一个铁CAD机体",
+  "advancement.psi:gold_assembly_pickup": "对得起金的重量",
+  "advancement.psi:gold_assembly_pickup.desc": "合成一个金CAD机体",
+  "advancement.psi:psimetal_assembly_pickup": "“Psi”天仙",
+  "advancement.psi:psimetal_assembly_pickup.desc": "合成一个Psi金属CAD机体",
+  "advancement.psi:ebony_assembly_pickup": "坚固且厚实",
+  "advancement.psi:ebony_assembly_pickup.desc": "合成一个乌金色CAD机体",
+  "advancement.psi:ivory_assembly_pickup": "光滑且性感",
+  "advancement.psi:ivory_assembly_pickup.desc": "合成一个象牙白CAD机体",
+  "advancement.psi:psidust": "……合成不了？",
+  "advancement.psi:psidust.desc": "合成你的第一份Psi粉末",
+  "advancement.psi:psimetal_pickup": "敬畏之合金",

```

```suggestion
  "advancement.psi:psimetal_pickup": "执畏熔金",
```
梗的是`The Alloy Of Awe 执法熔金`